src/usercontent/index.ts

```diff
@@ -11,10 +11,11 @@ function remoteRender(event) {
     const img = document.createElement("span"); // we'll mask it as an image
     img.id = "img";
 
-    const a = document.createElement("a");
+    const a: HTMLAnchorElement = document.createElement("a");
```

Comment:
alternatively, `document.createElement<HTMLAnchorElement>("a")` fwiw

Reply:
I actually wrote this by accident, TS seems to be able to infer the type based on the parameter